integrationTests/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "../base.tsconfig.json",
+    // NOTE: Everything except what you want compiled into JS should be in the exclusions.
+    "compilerOptions": {
+        "outDir": "./dist"
+    },
+    "exclude": [
+        "node_modules",
+        "gulpfiles",
```

Comment:
not needed here as we don't have any gulp files

Reply:
done